python/lsst/ip/isr/isrFunctions.py

```diff
@@ -219,8 +219,56 @@ def growMasks(mask, radius=0, maskNameList=['BAD'], maskValue="BAD"):
         fpSet.setMask(mask, maskValue)
 
 
-def maskITLEdgeBleed(ccdExposure, badAmpDict, itlEdgeBleedSatMinArea=10000,
-                     itlEdgeBleedSatMaxArea=100000,
+def getLargeSatMidPoint(fpCore):
+    """Get the number of cores and edges of saturated cores areas
+    in a saturated footprint.
+
+    Parameters
+    ----------
+    fpCore : `lsst.afw.detection._detection.Footprint`
+        Footprint of saturated core.
+
+    Returns
+    -------
+    trueSegments : int
+        Number of saturated cores in the footprint.
+    XMidPointBBox: `list` [`int`]
```

Comment:
All variables should lead with lower case (-> `xMidPointBBox`)

Reply:
This isn't a "bounding box" as such, and so I don't understand the variable name.